lib/omni_parse_client_ruby/parsers/vcf_parser.rb

```diff
@@ -42,7 +42,7 @@ def parse(p = {})
         prepare(p)
 
         self.last_request = base_url + parse_path
-        response = RestClient.post(last_request,
+        response = omni_post(last_request,
```

Comment:
Please change indentation to have beginning of the arguments in the same column, in every method invocation where needed.